process/transaction/baseProcess.go

```diff
@@ -137,6 +144,15 @@ func (txProc *baseTxProcessor) checkTxValues(tx *transaction.Transaction, acntSn
 		return process.ErrWrongTypeAssertion
 	}
 
+	txFee := txProc.economicsFee.ComputeFee(tx)
```

Comment:
this verification could be moved before line 134. I do not think cost.Cmp(big.NewInt(0)) is needed anymore.

Reply:
done